docs: update project-lifecycle.md for grammar

barriebyron opened the pull request:
Hi, Hyperledger community! 
I made a few grammatical edits as I try out the OSS contributions process to your project.

I always enjoy lifecycle processes and think these changes tighten up your message and process while adhering to grammar rules and best practices.

Other technical writers are looking for ways to grow their blockchain skills, so I am pulling together a list of projects they might want to contribute to.

barriebyron committed:
docs: update project-lifecycle.md for grammar

barriebyron commented on project-lifecycle.md:
```diff
@@ -7,24 +7,24 @@ nav_order: 6
-repository(s).
+projects may take the form of a new component (for example, a new repository) or
+may propose additions, deletions, or changes to an existing
+repository or repositories.
```
suggest we adhere to conventions as  https://developers.google.com/style/plurals-parentheses?hl=en

barriebyron commented on project-lifecycle.md:
```diff
@@ -7,24 +7,24 @@ nav_order: 6
 # Project Lifecycle
 
-The term "project" within Hyperledger will refer to a
+The term "project" within Hyperledger refers to a
```
Informing readers about project lifecycles require some verbs to use future tense. I suggest using present tense when possible, following https://developers.google.com/style/tense?hl=en

barriebyron commented on project-lifecycle.md:
```diff
@@ -43,40 +43,40 @@ and may go through several iterations.
--   identify committed development resources
--   identify initial maintainers
--   be vendor neutral
+-   Have a clear description
```
suggest capitalizing the first letter in a list item, following https://developers.google.com/style/lists?hl=en#numbered-lettered-bulleted-lists

barriebyron commented on project-lifecycle.md:
```diff
@@ -43,40 +43,40 @@ and may go through several iterations.
 [Hyperledger Github org](https://github.com/hyperledger)
 and optionally in [JIRA](https://jira.hyperledger.org), if requested. New
-features/capabilities should be handled through pull requests labeled
+features/capabilities must be handled through pull requests labeled
```
should is ambiguous

tkuhrt replied:
```suggestion
features or capabilities must be handled through pull requests labeled
```
For consistency with other changes you are making.

barriebyron reviewed (commented):
adding links to style guidance for the suggested changes, thanks for taking a look

tkuhrt commented:
Hi, @barriebyron : Thank you for submitting the pull request. Can you please follow the steps listed under [checks](https://github.com/hyperledger/tsc/pull/28/checks?check_run_id=5664439725) to correct the DCO sign-off requirement. In the meantime, I will take a look and provide some thoughts.

tkuhrt commented on project-lifecycle.md:
```diff
@@ -7,24 +7,24 @@ nav_order: 6
-projects may take the form of a new component (e.g. a new repository) or
-may propose additions, deletions or changes to an existing
-repository(s).
+projects may take the form of a new component (for example, a new repository) or
```
I believe `e.g.,` is okay

barriebyron replied:
```suggestion
projects may take the form of a new component (e.g., a new repository) or
```

tkuhrt commented on project-lifecycle.md:
```diff
@@ -43,40 +43,40 @@ and may go through several iterations.
 # Proposal
 
-Project Proposals shall be submitted to the [TSC](https://www.hyperledger.org/about/leadership)
+Project Proposals are submitted to the [TSC](https://www.hyperledger.org/about/leadership)
```
The verb here could even be `must be`, as this is the only way in which we accept project proposals today.

barriebyron replied:
```suggestion
Project Proposals must be submitted to the [TSC](https://www.hyperledger.org/about/leadership)
```

tkuhrt commented on project-lifecycle.md:
```diff
@@ -43,40 +43,40 @@ and may go through several iterations.
-components/modules, a repository will be created under the
+components and modules, a repository is created under the
 [Hyperledger Github org](https://github.com/hyperledger)
 and optionally in [JIRA](https://jira.hyperledger.org), if requested. New
```
```suggestion
. New
```

We no longer use Jira. Obviously not a grammar issue, but something that we should fix at some point.

tkuhrt commented on project-lifecycle.md:
```diff
@@ -43,40 +43,40 @@ and may go through several iterations.
 Entering *Incubation* does not guarantee that the project will
-eventually get to *Graduated* state. Projects may never get
+eventually get to the *Graduated* state. Projects may never get
 to *Graduated* state.
```
```suggestion
to the *Graduated* state.
```
Should we add `the` here to be consistent with previous change?

tkuhrt commented on project-lifecycle.md:
```diff
@@ -92,26 +92,25 @@ are in the *Graduated* phase.
-as to whether a project will move to or from the *Dormant* state upon request.
-If *Dormant* projects become re-activated, they will re-enter the *Incubation*
+Projects enter the *Dormant* state when the normal functions are
+suspended or slowed down for a period of time. The TSC decides to move project to or from the *Dormant* state upon request.
```
```suggestion
suspended or slowed down for a period of time. The TSC decides to move a project to or from the *Dormant* state upon request.
```

tkuhrt commented on project-lifecycle.md:
```diff
@@ -92,26 +92,25 @@ are in the *Graduated* phase.
-deprecate or not.
+that disagree with the request can make their case before the TSC. The
+TSC will consider all points of view and render a final decision to
+deprecate or not deprecate.
```
```suggestion
whether to deprecate.
```

tkuhrt commented on project-lifecycle.md:
```diff
@@ -92,26 +92,25 @@ are in the *Graduated* phase.
-TSC shall consider all points of view and render a final decision to
-deprecate or not.
+that disagree with the request can make their case before the TSC. The
+TSC will consider all points of view and render a final decision to
```
```suggestion
TSC will consider all points of view and render a final decision
```
in combination with my suggestion on the next line

barriebyron reviewed (commented):
thanks for the suggestions!

barriebyron committed:
Update project-lifecycle.md

barriebyron committed:
Apply suggestions from code review

thanks for looking at the suggestions and making this content even more awesome!

Co-authored-by: Tracy Kuhrt <tracy.a.kuhrt@accenture.com>

barriebyron commented:
closing in favor of https://github.com/hyperledger/tsc/pull/29
